### PR TITLE
Feature/e UI 3230 content changes

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3230

### Change description ###
Content changes from Caseworker to Staff

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by udaydenduluri33 — APPROVED** [review]
LGTMk

File: src/caseworker-ref-data/components/upload-info-partial-success/upload-info-partial-success.html [hunk 1]
```diff
@@ -25,9 +25,19 @@ <h3 class="govuk-heading-m">What went wrong </h3>
           </tbody>
         </table>
         <h3 class="govuk-heading-m">What you need to do</h3>
-        <p class="govuk-body">You’ll need to update the missing information.</p>
-        <p class="govuk-body">Once all the details in the template are correct, you’ll need to <a
-            routerLink="/caseworker-details">upload it again</a>.</p>
+        <ol>
+          <li class="govuk-body">
+            Check you out have the <a routerLink="/caseworker-details">latest version of the staff data upload
```
Comment (paul-graham) [inline src/caseworker-ref-data/components/upload-info-partial-success/upload-info-partial-success.html:30]: This doesn't read right. "Check you out have..."

I think it's a case of getting rid of "out".

**Review by paul-graham — CHANGES_REQUESTED** [review]

File: src/caseworker-ref-data/components/upload-info-partial-success/upload-info-partial-success.html (hunk 1 above)
Comment (kuda1992) [reply src/caseworker-ref-data/components/upload-info-partial-success/upload-info-partial-success.html:30]: @paul-graham Definitely doesn't read right now that I am reading it again

**Review by paul-graham — APPROVED** [review]

**Pull request merged**